Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/web/TaboolaWebWrapperViewModel.kt

```diff
@@ -32,14 +38,41 @@ class TaboolaWebWrapperViewModel: ViewModel() {
      * This method is just a quick way to load an example page simulating customer layout.
      * Inside the asset file there are Javascript tags Taboola targets with its content.
      */
-    fun loadWebViewContent(webView: WebView, properties: PlacementInfo.Properties, context: Context) {
+    fun loadWebViewContentWidget(
+        webView: WebView,
+        properties: PlacementInfo.WidgetProperties,
+        context: Context
+    ) {
         val BASE_URL = "https://example.com"
         val HTML_CONTENT_FILE_TITLE = "sampleContentPage.html"
-        var htmlContent: String? = null
         try {
-            htmlContent = AssetUtil.getHtmlTemplateFileContent(context, HTML_CONTENT_FILE_TITLE)
-            htmlContent = htmlContent.replace("<PLACEMENT>", properties.placementName)
-            htmlContent = htmlContent.replace("<MODE>", properties.mode)
+            val htmlContent: String =
+                AssetUtil.getHtmlTemplateFileContent(context, HTML_CONTENT_FILE_TITLE)
+                    .replace("<PLACEMENT>", properties.placementName)
```

Comment:
@voltfmanSlava  - Worth to put these string literals into constants ("<PLACEMENT>")

Reply:
Done